getCourses.py

```diff
@@ -0,0 +1,37 @@
+from flask import Flask, request
+import requests
+import json
+app = Flask(__name__)
+@app.route('/',methods = ['GET'])
+URL = 'http://api.culpa.info/courses/department_id/7'
+
+def getAllData():
+    return null
+def getProfessors():
+    r = requests.get(URL)
+    json_data = json.loads(r.text)
+    array = [len(json_data['courses'])]
+    for i in range(len(json_data['courses'])):
+        array[i] = str(json_data['courses'][i]['name'])
```

Comment:
Does this work? `array` is a list of length 1, but here we're assuming it's the same length as `json_data["courses"]`.

Ideally, this should be a [list comprehension](http://www.secnetix.de/olli/Python/list_comprehensions.hawk):

````Python
return [course["name"] for course in json_data["courses"]] 
````

Or if we're feeling fancy, we can also use my favorite utility belt library [toolz](https://github.com/pytoolz/toolz) and [`pluck`](http://toolz.readthedocs.io/en/latest/api.html#toolz.itertoolz.pluck)